tests/input_files/xml/preconditioner/ifpack.xml

```diff
@@ -0,0 +1,9 @@
+<ParameterList name="Ifpack">
+  <!-- General settings for Ifpack preconditioners -->
+  <Parameter name="Preconditioner type" type="string" value="ILU"/>
+  <Parameter name="Overlap" type="int" value="0"/>
+  <!-- Level-of-fill setting for Ifpack ILU preconditioner -->
```

Comment:
The parameter name seems to be self-explaining, so no need for this comment. One needs to look at the Ifpack documentation anyways for further information on possible parameters.
```suggestion
```